Source/dom/models/Override.js

```diff
@@ -14,9 +14,10 @@ export class Override extends WrappedObject {
     if (!this.__symbolInstance) {
       return undefined
     }
+		this.__symbolInstance.sketchObject.ensureDetachHasUpdated()
     return toArray(
       this.__symbolInstance.sketchObject.overrideContainer().flattenedChildren()
-    ).find((x) => x.availableOverride() == this.sketchObject)
+    ).find((x) => x.availableOverride().overridePoint().name().isEqual(this.sketchObject.overridePoint().name()))
```

Comment:
We don't have the same caching as we had before so the underlying objects could be different and the equality check could fail. Checking the override point name is safer

... but also we need to use `isEqual` rather than `==` 😖